index.html

```diff
@@ -4381,6 +4377,75 @@ <h3>Parsing content</h3>
       [[RFC2048]] and [[RFC2046]].
     </p>
   </section>
+  <section> <h3>Things that users should be made aware of when using NFC</h3>
+    <p>
+      This section details some of the things that users ought to be aware
+      of when using NFC. It is recommendated that implementations help educate
+      the users of given facts before or when related NFC actions are performed. 
+    </p>
+    <h4>Data that is read is shared with site</h4>
+    <p>
+      When a site has access to read NFC content, then the data of the scanned
+      tags is shared with the site, in a similar way to uploading files and images.
+      The user has to trust that the site handles this data properly and in the
+      intended manner.
+    </p>
+    <h4>A site may modify and overwrite data of tags that are not made read-only</h4>
+    <p>
+      Deployed NFC solutions, like tags in stores etc, should always be made
+      read-only in order to ensure they are not modified by mistake or as part
+      of a malicious act.
+    </p>
+    <p>
+      Private tags and stickers are often unlocked (writable) from the factory
+      and the user should be aware that such tags might be overwritten/modified
+      by scanning them.
+    </p>
+    <h4>Reading a fixed (e.g. mounted) tag may expose reading location</h4>
+    <p>
+      A fixed tag may encode its ID or location in the data, meaning that reading
+      it exposes that information to the site, which then can deduct the location
+      the read took place. That combined with being logged into a service, can
+      share your location data with the site.
```

Comment:
if the site planted the id or the tag  - it has to have prior knowledge about the tag in order to determine location

Reply:
that is implied. I am trying to keep this simple

Reply:
That's an important distinction, since this threat is not generic: in the vast majority of the cases sites won't deduct location by reads. The precondition is important for the attack.